zerver/tests/test_events.py

```diff
@@ -847,7 +847,7 @@ def test_realm_bots_non_admin(self):
         user_profile = get_user_profile_by_email(email)
         self.assertFalse(user_profile.is_realm_admin)
         result = fetch_initial_state_data(user_profile, None, "")
-        self.assert_length(result['realm_bots'], 0)
+        self.assert_max_length(result['realm_bots'], 0)
```

Comment:
Oh yes, of course. I did not take notice of the zero. I'll modify it. :) Thank you.